backend/src/main/java/ch/zuehlke/fullstack/hackathon/RestTemplateFactory.java

```diff
@@ -0,0 +1,37 @@
+package ch.zuehlke.fullstack.hackathon;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class RestTemplateFactory {
```

Comment:
I would take this directly into InsightClient, as this is highly related to the insight api